packages/core/src/card/Card.css

```diff
@@ -1,19 +1,66 @@
 /* Styles applied to the root element */
 .saltCard {
-  box-shadow: var(--saltCard-boxShadow, var(--salt-overlayable-shadow));
-  border-width: var(--salt-size-border);
+  border-width: var(--saltCard-borderWidth, var(--salt-size-border));
   border-style: var(--salt-container-borderStyle);
-  padding: var(--saltCard-padding, var(--salt-spacing-300));
+  padding: var(--saltCard-padding, var(--salt-spacing-200));
+  position: relative;
 }
 
 .saltCard-primary {
   background: var(--saltCard-background, var(--salt-container-primary-background));
   border-color: var(--salt-container-primary-borderColor);
+  --card-accentColor: var(--salt-container-primary-borderColor);
```

Comment:
This should follow our standards on token naming. Camel case is only used when the token refers to a native CSS attribute e.g. borderColor. In this can accentColor isn't  native attribute so wouldn't use camel case.

Reply:
Yes I was having trouble with this I mentioned it somewhere but can't remember now.

I thought it should be `--card-accent-color` but that was failing lint:style.

Reply:
ohh it is because there is a css property called `accent-color` whereas there isn't one called `text-color`

Reply:
Ah you're right there's a css property called `accent-color`. This is a tricky one and breaks our rules :(